.ci/pytorch/build.sh

```diff
@@ -301,6 +301,11 @@ else
     fi
     pip_install_whl "$(echo dist/*.whl)"
 
+    if [[ "$BUILD_ENVIRONMENT" == *xpu* ]]; then
+      echo "Checking that xpu is compiled"
+      python -c 'import torch; exit(0 if torch.xpu._is_compiled() else 1)'
```

Comment:
It is better to check the return value and then print an error message if `torch.xpu._is_compiled` is `False`.

```bash
python -c 'import torch; exit(0 if torch.xpu._is_compiled() else 1)'
if [ $? -eq 0 ]; then
    echo "XPU support is compiled in"
else
    echo "XPU support is NOT compiled in"
    exit 1
fi
```

Reply:
The original method can work with `set -e`, but I can update it with this safer method.